spec.toml

```diff
@@ -0,0 +1,3 @@
+server_url = "http://localhost:8000"
+faucet_pem = "faucet.pem"
```

Comment:
We should probably provide a valid faucet, i.e., the `id1` identity we're using for dev.

I would like users and devs to be able to run those tests without doing anything more than build/test.

Reply:
@hansl Pinging you here following our 1:1.

Reply:
@fmorency the idea is that the faucet.pem file is copied from the pem file devs use, but we don't commit that file to the repository.

Reply:
I understand. 

However, I believe that we should provide sane defaults to run the spec tests against a default many-framework config, without any user intervention except running `cargo test`.

We need to discuss this with @hansl

Reply:
I think that (for now) this is safe to ignore. `many-framework` should provide a `spec.toml` file for its own tests.

We won't be able to have sane defaults for private keys anyway (and I'd rather not).